web/src/package.json

```
@@ -70,9 +70,9 @@
     "@date-io/luxon": "1.0.1",
     "@date-io/moment": "^0.0.2",
     "@hot-loader/react-dom": "^16.7.0",
```

This one should match the react version too I believe
```suggestion
     "@hot-loader/react-dom": "16.8.6",
```